src/azure-cli/azure/cli/command_modules/serviceconnector/_help.py

```diff
@@ -397,3 +402,299 @@ def get_source_display_name(sourcename):
                server_params=server_params,
                registry_params=registry_params,
                source_display_name=source_display_name)
+
+
+source = RESOURCE.Local
+connection_id = (
+    '/subscriptions/{subscriptionId}/resourcegroups/{resourceGroupName}/'
+    'providers/Microsoft.ServiceLinker/locations/{location}/connectors/{connectorName}')
+source_display_name = 'Service Connector'
+helps['connection'] = """
+    type: group
+    short-summary: Commands to manage {} local connections
```

Comment:
Briefly explain what's a `local connection`, and tell users to use `az webapp/containerapp/spring connection` if they want to manage connections for compute services

Reply:
Ok